pkgs/applications/virtualization/runc/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, go-md2man
+, go, pkgconfig, libapparmor, apparmor-parser, libseccomp }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "runc-${rev}";
+  rev = "cc29e3dd"; # this version is used by docker
```

Comment:
Working packaging is more important then strictly sticking to released versions. IOW, I was just making a comment. Not a blocker.